sys/net/gnrc/pktdump/Kconfig

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2020 Freie Universitaet Berlin
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+menuconfig KCONFIG_MODULE_GNRC_PKTDUMP
+    bool "Configure GNRC_PKTDUMP"
+    depends on MODULE_GNRC_PKTDUMP
+    help
+        Configure the GNRC_PKTDUMP using Kconfig.
+
+if KCONFIG_MODULE_GNRC_PKTDUMP
+
+config GNRC_PKTDUMP_MSG_QUEUE_SIZE_EXP
+    int "Queue size (as exponent of 2^n)"
+    default 4
```

Comment:
Shouldn't this be 3, given that `GNRC_PKTDUMP_QUEUE_SIZE` is currently 8?

Reply:
Yes. I shall correct it